database/migrations/2023_09_06_072008_create_users_table.php

```diff
@@ -16,9 +16,7 @@ public function up(): void
             $table->string('name');
             $table->string('email')->unique();
             $table->string('provider_id')->nullable();
-            $table->timestamp('email_verified_at')->nullable();
-            $table->string('password')->nullable();
-            $table->rememberToken();
+            $table->string('avatar')->nullable();
```

Comment:
This Migration has been done.
Please move these into the `database/migrations/2023_09_12_144631_add_birthday_to_users.php` migration which has not been merged yet